show/main.py

```diff
@@ -796,6 +810,68 @@ def alias(interfacename):
 
     click.echo(tabulate(body, header))
 
+#
+# 'breakout' group ###
+#
+@interfaces.group(invoke_without_command=True)
+@click.pass_context
+def breakout(ctx):
+    """Show interface breakout"""
+
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    ctx.obj = {'db': config_db}
+
+    curBrkout_tbl = config_db.get_table('BREAKOUT_CFG')
+    if ctx.invoked_subcommand is None:
+
+        # Get HWSKU and Platform information
+        hw_info_dict = get_hw_info_dict()
+        platform = hw_info_dict['platform']
+        hwsku = hw_info_dict['hwsku']
+
+        # Get port capability from platform and hwsku related files
+        platformDict = readJsonFile("{}/{}/{}".format(PLATFORM_ROOT_PATH, platform, PLATFORM_JSON))['interfaces']
+        hwskuDict = readJsonFile("{}/{}/{}/{}".format(PLATFORM_ROOT_PATH, platform, hwsku, HWSKU_JSON))['interfaces']
+
+        # Update 'hwskuDict' to 'PlatformDict'
+        for port_name in platformDict.keys():
+            platformDict[port_name].update(hwskuDict[port_name])
+            platformDict[port_name].update(curBrkout_tbl[port_name])
+
+        # Sorted keys by name in natural sort Order for human readability
+        parsed = OrderedDict((k, platformDict[k]) for k in natsorted(platformDict.keys()))
+        click.echo(json.dumps(parsed, indent=4))
+
+# 'breakout current-mode' subcommand ("show interfaces breakout current-mode")
+@breakout.command('current-mode')
+@click.argument('interface', metavar='<interface_name>', required=False, type=str)
+@click.pass_context
+def currrent_mode(ctx, interface):
+    """Show interface breakout current-mode"""
+
+    config_db = ctx.obj['db']
+
+    header = ['Interface', 'Current Breakout Mode']
+    body = []
+
+    curBrkout_tbl = config_db.get_table('BREAKOUT_CFG')
+
+    if not curBrkout_tbl:
+        click.echo("Breakout table is not present in Config DB")
+        return
+
+    # Show current Breakout Mode of user prompted interface
+    if interface is not None:
+        body.append([interface, str(curBrkout_tbl[interface]['brkout_mode'])])
+        click.echo(tabulate(body, header, tablefmt="grid"))
+        return
+
+    # Show current Breakout Mode for all interfaces
+    for name in curBrkout_tbl.keys():
+        body.append([name, str(curBrkout_tbl[name]['brkout_mode'])])
+    click.echo(tabulate(body, header, tablefmt="grid"))
```

Comment:
Please nature sort the interface names here.

Reply:
done